helm/production-values.yaml

```diff
@@ -0,0 +1,5 @@
+environment: production
+containers:
+  service:
+    registry: 'minesupport.azurecr.io/'   

```

Comment:
You could have the `/`in the deployment yaml in the same way you have the `:` to avoid the trailing slash on the registry name

Reply:
The reason I didn't was so I could default to using a local image if a registry wasn't supplied.  

So if I did that with no registry I'd get /imageName instead of imageName.  I believe Helm has conditional statements so will explore those to refactor further.

Reply:
Updated template to use conditional to determine whether to insert registry and "/".